src/manager/kiwix-manage.cpp

```diff
@@ -56,20 +56,43 @@ void show(kiwix::Library* library, const std::string& bookId)
   std::cout << std::endl;
 }
 
+/* Print correct console usage options */
 void usage()
 {
-  cerr << "Usage:" << endl;
-  cerr << "\tkiwix-manage LIBRARY_PATH add ZIM_PATH "
-          "[--zimPathToSave=../content/foobar.zim] [--current] "
-          "[--url=http://...metalink]"
-       << endl;
-  cerr << "\tkiwix-manage LIBRARY_PATH show [CONTENTID1] [CONTENTID2] ... "
-          "(show everything if no param.)"
-       << endl;
-  cerr << "\tkiwix-manage LIBRARY_PATH remove CONTENTID1 [CONTENTID2]" << endl;
+  std::cout << "Usage:" << std::endl
+            << "\tkiwix-manage LIBRARY_PATH add ZIM_PATH [OPTIONS]" << std::endl
+            << "\tkiwix-manage LIBRARY_PATH remove ZIM_ID [ZIM_ID]..." << std::endl
+	    << "\tkiwix-manage LIBRARY_PATH show [ZIM_ID]..." << std::endl
+            << std::endl
+    
+            << "Purpose:" << std::endl
+            << "\tManipulating Kiwix library XML file"
+            << std::endl << std::endl
+
+            << "Arguments:" << std::endl
+            << "\tLIBRARY_PATH\tis the XML library file path."
+            << std::endl << std::endl
+            << "\tACTION\t\tis the pre-defined string to specify the action to run on the XML library file."
+	    << std::endl << std::endl
+            << "\t\t\tMust be one of the following values:" << std::endl
+            << "\t\t\t* add: add a ZIM file to the library" << std::endl
+            << "\t\t\t* remove: remove a ZIM file from the library" << std::endl 
+            << "\t\t\t* show: show the content of the library"
+	    << std::endl << std::endl
+            << "\tZIM_ID\t\tZIM file unique ID"
+            << std::endl << std::endl
+            << "\tOPTIONS\t\tCustom options for \"add\" action:" << std::endl
+            << "\t\t\t--zimPathToSave=CUSTOM_ZIM_PATH to replace the current ZIM file path" << std::endl
+            << "\t\t\t--url=HTTP_ZIM_URL to create an \"url\" attribute for the online version of the ZIM file" << std::endl
+            << "\t\t\t--current to specify the ZIM to open per default in this library" << std::endl
```

Comment:
There is no current option.
It has been removed 6 months ago (October 2018). I should have remove it from the usage string also.

Reply:
@mgautierfr good, will remove it and merge. Thx.

Reply:
@mgautierfr removed